### PR TITLE
UI

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the following was done: 

1. App.rb and Main.rb file was created
2. A menu was created
3. Users and books were added
4. Searches were done

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+def rental_date
+  print 'Date (YYYY/MM/DD): '
+  gets.chomp
+end
+
+def create_and_add_rental(rentals, date, book, person)
+  rental = Rental.new(date, book, person, person.class.to_s)
+  rentals << rental
+end
```
Comment (OyePriscilla) [inline app.rb:117]: - [ ] Please kindly take note that I was unable to create a rental, the screenshot below is the error message that I got `create_rental: Undefined variable or method **get_rental_date.**` 👇 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @dvcorreia1984 👏 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Create a UI for school library app ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (dvcorreia1984) [reply app.rb:117]: Hi, this upload does not show on my side. But I will have a look anyway!

Comment (dvcorreia1984) [reply app.rb:117]: Ok, found the error. Thanks!

**Review by cynthiainga — APPROVED** [review]
Hi @dvcorreia1984, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Requested change is made ✔️ 
- [ ] The app works as expected ✔️ 
- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (OyePriscilla) [reply app.rb:117]: @dvcorreia1984 , I'm so sorry for late response, kindly take note that you should have used the '@' tag in order for me to be notified about this message . I'm glad that you've found and resolved the error pointed out 👍. Thanks.

**Pull request merged**